otx/algorithms/action/adapters/mmaction/data/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""OTX Algorithms - Action Classification Dataset."""
```

Comment:
Is there any chance to include other types included in mmaction (i.e. Video Classification, Video Recognition, Action Recognition) ? If yes, how about change this line for further scenario ?

Reply:
Needs to make unified word / sentence for using mmaction. 

In this [link](https://github.com/openvinotoolkit/training_extensions/pull/1328/files#diff-bf927e97fa4cb305e8cc97a2eab9de9b54ce7fdcd059f3efb89496974df5d9ceR1), 'Action Recognition' is used

Reply:
Very good point, thnk you for asking Sungman. In my knowledge, we use the term "Video Recognition" as Video Classification + Video Detection(Spatio and temporal)
And Video Classification contains Person video classification(known as action classification), and Non-person video classification. Similarly Video detection contains Person video detection(known as action detection), and Non-person video detection
This PR is for adding action classification first. Action detection will be added soon. Non-person video classification and Non-person video detection will be added, but it will require longer time
To be frankly, the module added in this PR can be used other tasks, the name for each module will be assign after merging action detection

Reply:
Understood, I just want to recommend changing the word `Action Classfication` to `Video Recognition` for our future path.

Reply:
If it is used in both action detection and action classification. At current stage, the usage of this dataset is not determined. I will set the correct name when I add action detection task.